app/views/layouts/_header.html.erb

```diff
@@ -68,6 +68,22 @@
 </div>
 
 <script type="text/javascript">
+  document.addEventListener("DOMContentLoaded", function () {
+    var gettingStartedLink = document.querySelector("#getting-started-dropdown");
+    gettingStartedLink.addEventListener("click", function () {
+      if (gettingStartedLink.classList.contains("rotate180")) {
+        gettingStartedLink.classList.remove("rotate180");
+      }
+      else{
+        gettingStartedLink.classList.toggle("rotate180");
+      }
+    });
```

Comment:
`toggle` method itself add class if not present and remove if present, so it's not required to manually remove if contains
Ref - https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle